src/Shared/Components/Plugin/service.tsx

```diff
@@ -36,12 +36,14 @@ export const getPluginsDetail = async ({
     pluginIds,
     signal,
     shouldShowError = true,
+    parentPluginIdentifiers,
 }: PluginDetailServiceParamsType): Promise<Pick<GetPluginStoreDataReturnType, 'pluginStore'>> => {
     try {
         const payload: PluginDetailPayloadType = {
             appId,
             parentPluginId: parentPluginIds,
             pluginId: pluginIds,
+            parentPluginIdentifier: parentPluginIdentifiers ? `${parentPluginIdentifiers}` : null,
```

Comment:
This handling should be done inside getUrlWithSearchParams, even during dev of plugin versioning there was a consensus with BE regarding the format

Reply:
By rest convention, we pass the value in array multiple times in the query params and getUrlWithSearchParams sticks to the same. In case we intend to break the existing functionality it would have impact across the APIs